Storage/Storage/Model/CustomerSearchResult+CoreDataProperties.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+import CoreData
+
+
+extension CustomerSearchResult {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<CustomerSearchResult> {
+        return NSFetchRequest<CustomerSearchResult>(entityName: "CustomerSearchResult")
+    }
+
+    @NSManaged public var customerID: Int64
+    @NSManaged public var id: NSSet?
```

Comment:
Similar to `Customer`, this should be named for what's in the search results, i.e.
```suggestion
    @NSManaged public var customers: Set<Customer>?
```

E.g. [ProductSearchResults+CoreDataProperties](https://github.com/woocommerce/woocommerce-ios/blob/de959a276be9c197d97855c482ab242a573bda9e/Storage/Storage/Model/ProductSearchResults%2BCoreDataProperties.swift#L13)